README.md

```diff
@@ -361,6 +362,70 @@ To stop receiving events from a stream, pass the `Subscription` object you got w
 client.unsubscribe(sub);
 ```
 
+<a name="data-unions"></a>
+## Data Unions
+This library provides functions for working with Data Unions. To get a DataUnion client instance, call
```

Comment:
@kare can you spread these out, like 

This library provides functions for working with Data Unions. 

To get a DataUnion client instance, call:
```
client.dataUnionClient(mainnetAdminPrivateKey, sideChainAdminPrivateKey)
```

To deploy a new DataUnion, call:

```
dataUnionClient.deployDataUnion(name, adminAddress, adminFeeFractionWei, agents)
```

To get an existing instance of Data Union, call:
```
dataUnionClient.dataUnionFromName(name)
```